### DISCUSSION TITLE
Throws this error

### DISCUSSION THREAD

**Discussion by rustyprophet213** [GiorgioBrux/nitro-sniper-enhanced#164] [Q&A]
![Capture454](https://user-images.githubusercontent.com/74872908/137637414-97dba9d8-8c79-4e44-ab22-f0f72be359b4.PNG)
also yea i'm no coding pro so i have no clue how to handle this error

**Comment by GiorgioBrux** [comment]
are you on node 14.x?

**Reply by rustyprophet213** [reply]
not running it locally on my system i'm using the sniper on repl.it so i just imported the entire project with the github link not sure about the node version also how do i check it  tho

**Reply by GiorgioBrux** [reply]
Make sure you are using the run button. If it still doesn't work, create an empty (or if doesn't let you, one with a random character) env variable and try agian.
